### DISCUSSION TITLE
Parcels particles repeated release - stop  - repeated release

### DISCUSSION THREAD

**Discussion by SeanBensadon** [Parcels-code/Parcels#1623] [General]
Hi,
I am having some trouble trying to get multiple repeat releases and stop releases of particles using Parcels.
I want to be able to repeatedly release particles for a specific duration (e.g. particles released every hour for 12 hours) then stop the release of particles for a period of time, and then continue the repeated release of particles again for another time duration (e.g. every hour for 12 hours), and then stop again. 
I haven't found a way of achieving this yet and wanted to know if this was possible or whether anyone had managed to overcome this issue?.
Thank you for the help

**Comment by VeckoTheGecko** [comment]
Hi @SeanBensadon ,
In this case you're best off assigning each particle a time rather than using the `repeatdt` parameter. This is covered in our ["Delayed starts"](https://docs.oceanparcels.org/en/latest/examples/tutorial_delaystart.html#Assigning-each-particle-its-own-time) tutorial in the section "Assigning each particle its own time". That will give you a working code example to get started with.

Hope that helps! 😁
